lib/routes.js

```diff
@@ -54,10 +54,17 @@ const create = async function (server, { api, basedir, cors, vhost, handlers, ex
             options.handler = handler;
 
             if (Utils.canCarry(method)) {
-                options.payload = {
-                    allow: operation.consumes || api.consumes
-                };
+                if (options.payload) {
+                    if (!options.payload.allow) {
```

Comment:
I don’t think this should be settable in two places. Should always use the swagger definition of consumes